src/command_helpers.rs

```diff
@@ -18,13 +18,15 @@ use crate::{Error, ErrorKind, Object};
 pub(crate) struct CargoOutput {
     pub(crate) metadata: bool,
     pub(crate) warnings: bool,
+    pub(crate) debug: bool,
 }
 
 impl CargoOutput {
-    pub(crate) const fn new() -> Self {
+    pub(crate) fn new() -> Self {
         Self {
             metadata: true,
             warnings: true,
+            debug: std::env::var("CC_ENABLE_DEBUG_OUTPUT").is_ok(),
```

Comment:
```suggestion
            debug: std::env::var_os("CC_ENABLE_DEBUG_OUTPUT").is_some(),
```

let's use var_os to accept non utf-8 characters, we don't care about the content only the present of the env variable.

Reply:
yep makes sense